modules/statistics/test/TestPlan.md

```diff
@@ -30,18 +30,11 @@
 24. Returning to the behavioural statistics page, verify that if you do not have the access_all_profiles permission you should not be able to click through to forms for candidates from other sites.
 25. Check the the breadcrumbs from Completion stats page back to BVL stats tab within Statistics appear and allow you to use the "back" button to previous pages (future feature).
 
-### Reliability Statistics
-26. Click on the Reliability Statistics tab. Try using the reliability filter. Check that you can filter by site and by projects, separately.
-27. For one of the reliability modules, check that the data in each column makes sense in comparison to data in the database.
-28. There should only be one reliability table on this page right?
-
-### MRI Statistics
-29. Click on the MRI statistics tab. Try using the filter query. Filter by site and by project separately to ensure they are both working.
-30. Look at the first table on this page (the first row is 'Complete' in MRI parameter form). Do the numbers in each cell match what is in the database. Should these columns have headers?
-31. Looking at the MRI integrity statistics, do the numbers in the table make sense in comparison to what's stored in the database?
-32. Try following the "Click Here for breakdown per participant" links. Do they take you to the appropriate page?
-33. On the breakdown page you just loaded, are the incomplete candidates listed correct in comparison to what's listed as incomplete in the database?
-34. Try clicking on one of the incomplete candidate links. Does it take you to the correct page in the imaging browser?
-35. Returning to the main MRI statistics page, try using the scan breakdown filter. Filter by each type of scan. Does it work?
-36. For one of the scan breakdowns, does the information in the table make sense in comparison to what is in the database? Does the % complete appear to be being calculated correctly.
-37. Does the grand total row make sense with the data that is listed in the table?
+### Imaging Statistics
+26. Click on the Imaging Statistics tab. Try using the Site and Project filters. Does it change the data? Double check that the data represented is consistent with the data in the database.
+27. For the 1st table, check that the Total column adds up correctly.
+28. Check that Select All, as well as different combinations of scan types adjust the table accordingly.
+29. Check that the data in the Breakdown by Scan Type table is accurate.
+30. Check that show/hide visit labels works.
+31. Check that changing the breakdown adjusts the table accurately.
+32. Verify site and grand totals.
```

Comment:
@driusan these were removed in 16.04 by me when MRI stats was IBIS specific. after the mini-revamp we removed the feature but I never updated the testplans